hist/histv7/inc/ROOT/RHistImpl.hxx

```diff
@@ -120,10 +131,7 @@ public:
 
    /// Get a AxisIterRange_t for the whole histogram, possibly restricting the
    /// range to non-overflow bins.
-   ///
-   /// \param withOverUnder - specifies for each dimension whether under and
-   /// overflow should be included in the returned range.
-   virtual AxisIterRange_t GetRange(const std::array<Hist::EOverflow, DIMENSIONS> &withOverUnder) const = 0;
+   virtual AxisIterRange_t GetRange() const = 0;
```

Comment:
The doc comment needs updating to reflect the API change.

Reply:
Yes, I know, I'm working on it.